Scripts/Python/xLinkingBookGUIPopup.py

```diff
@@ -527,9 +527,10 @@ def IShowBookNoTreasure(self):
                 gLinkingBook = ptBook(bookdef,self.key)
                 gLinkingBook.setSize( width, height )
                 # make sure there is a cover to show
-                if not showOpen:
-                    if not self.IsThereACover(bookdef):
-                        showOpen = 1
+                if not showOpen and not self.IsThereACover(bookdef):
+                    showOpen = 1
+                else:
+                    showOpen = 0
```

Comment:
```suggestion
                if showOpen and self.IsThereACover(bookdef):
                    showOpen = 0
```

Reply:
Also Line 867 ` if idx > 0:` should be ` if idx >= 0:`
Anything other than -1 is true for string find